faq.md

```diff
@@ -5,69 +5,80 @@ permalink: "faq.html"
 
 In case the above did not work, evaluate the following in the JavaScript console:
 
-    require('amber/helpers').popupHelios()
+`require('amber/helpers').popupHelios()`
 
     
-### Q: What version is the amber website running?
-
-1.    Open IDE at http://amber-lang.net/.
-2.    Go to Workspace tab.
-3.    Print-it ``Smalltalk version``.
-4.    
-
-### How do I install the latest Amber version?
-
-'Latest' often means the 'latest stable'. If this is the case go for
+#### Q: What version is the amber website running?
 
-      npm -g install amber-cli
+1.    Open IDE at [http://amber-lang.net/](http://amber-lang.net/).
+2.    Go to the Workspace tab.
+3.    Print or inspect the expression ``Smalltalk version``.
 
-If you mean really the prereleases, 
+#### How do I install the latest Amber version?
 
-      npm -g install amber-cli@bleedingedge 
+For the **latest stable**, go for
 
-should install the last prerelease (but not the stable version even if it is newer).
+`npm -g install amber-cli`
 
-Use `npm info amber-cli` to find out which version is latest and which one is bleedingedge, if you are unsure.
+If you want the **latest prerelease**, go for
 
+`npm -g install amber-cli@bleedingedge`
 
-### How do I update Amber?
+Note that bleedingedge will *not* install the stable version even if it is newer.
 
--  ``npm install -g amber-cli``  will get it fixed for all new projects and 
+If you want to be sure which version is latest and which one is bleedingedge use:
+`npm info amber-cli` and you will find them in the `dist-tags` section.
 
--  ``npm update amber-dev`` in every project dir will fix it for current projects
+#### How do I update Amber?
 
+1. ``npm install -g amber-cli``  will get it fixed for all new projects and 
 
-### How do I get the text value of an input field?
+2.  ``npm update amber-dev`` in every project dir will fix it for current projects
 
-Assuming I have a field with an id of #field1, how do I get the text value of it?
+3. `bower install` in every project dir will fix it for current projects
 
-Answer:
 
-   '#field1' asJQuery val
+#### How do I get the text value of an input field?
 
-Note:
+If you have a field with an id of #field1, the way to get it's value is `'#field1' asJQuery val`.
 
-All Javascript methods in jQuery are available to you this way.
+Note: _All_ JavaScript methods in jQuery are available to you this way.
 
 Multi parameter functions are then mapped like:
 
-    aQuery.sampleFunc (a,b,c);
+`aQuery.sampleFunc(a,b,c);`
 
-    aQuery sampleFunc: a and: b and: c.
+ `aQuery sampleFunc: a andThis: b andThat: c.`
 
-The ``and:`` can be whatever. The only thing that matters is the first part of the keyword selector which has to match the function name.
+The ``andThis:andThat:`` part can be any selectors. The only thing that matters for JavaScript and Amber is the first part of the keyword selector which has to match the JavaScript function name.
```

Comment:
`sampleFunc:andThis:andThat:` is the complete selector, if I am not mistaken.
Therefore the wording of the first sentence is technically incorrect.

Maybe it should read:

The ``andThis:andThat:``part of the Smalltalk selector can be named any way you want.
For JavaScript functions the names of those keyword are ignored by Amber except for the first one which has to match the JavaScript function name.